.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,7 +4,7 @@
 
 ### All Submissions:
 
-- [ ] Have you followed the guidelines in our [Contributing](https://github.com/usnistgov/OSCAL/blob/master/CONTRIBUTING.md) document?
+- [ ] Have you followed the guidelines in our [Contributing](https://github.com/usnistgov/OSCAL/blob/master/CONTRIBUTING.md) document? Have you selected the right base branch?
```

Comment:
Suggest changing:
"Have you selected the right base branch?" to 
"Have you selected the correct base branch per [Contributing](https://github.com/usnistgov/OSCAL/blob/master/CONTRIBUTING.md) guidance?"

Reply:
Done in https://github.com/usnistgov/OSCAL/pull/1094/commits/f4149027fd74498a0ff649fd603860fa2a1c8180.